### PR TITLE
Removes world users hooks + eliminate its usage to seated users

### PR DESCRIPTION
**Pull request by mike-lvov**
Removes world users hooks + eliminate its usage to seated users

### CODE REVIEW

File: src/components/atoms/UserAvatar/UserAvatar.tsx [hunk 1]
```diff
@@ -52,7 +51,8 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
   showStatus,
   size,
 }) => {
-  const { isOnline } = useIsOnline(user?.id);
+  // @debt temporarily disable is online functionality
+  const isOnline = false;
```
Comment (mike-lvov) [inline src/components/atoms/UserAvatar/UserAvatar.tsx:55]: It didn't seem to work properly before. Temporarily disable it

File: src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx [hunk 2]
```diff
@@ -15,10 +13,7 @@ export interface RunTabUserInfoProps {
 }
 
 export const RunTabUserInfo: React.FC<RunTabUserInfoProps> = ({ user }) => {
-  const { userLocation } = useWorldUserLocation(user?.id);
-  const userLastSeenIn = userLocation?.lastVenueIdSeenIn;
-
-  const location = userLastSeenIn ? Object.keys(userLastSeenIn)?.[0] : "";
+  const location = user.lastVenueIdSeenIn;
```
Comment (mike-lvov) [inline src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx:16]: OOS improve the view of the location

File: src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx [hunk 3]
```diff
@@ -123,13 +112,7 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
...
 
       <div className="PosterPreview__authorBox">
-        {presenterUser && (
-          <UserProfilePicture
-            containerClassName="PosterPreview__avatar"
-            user={presenterUser}
-            showStatus
-          />
-        )}
+        {authorName}
```
Comment (mike-lvov) [inline src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx:115]: This logic was written by an opensource contributor. If we want to use this logic in the future, we would change the implementation to add the presenter when adding the data

File: src/hooks/users/useWorldUsersById.ts [hunk 4]
```diff
@@ -29,16 +27,3 @@ export const useWorldUsersById = () => {
...
-export const useWorldUsersByIdWorkaround = () => {
-  const {
-    worldUsersById: _worldUsersById,
-    isWorldUsersLoaded,
-  } = useWorldUsersById();
-
-  const worldUsersById: Partial<Record<string, User>> = _worldUsersById;
-
-  return { worldUsersById, isWorldUsersLoaded };
-};
```
Comment (mike-lvov) [inline src/hooks/users/useWorldUsersById.ts:44]: Given that we will remove this from the codebase, I decided to simplify this task and to leave only one `wrong` implementation

**Review by dmitrymarchuk — APPROVED** [review]
LGTM besides some qustions/suggestions

File: src/components/templates/AnimateMap/bridges/CloudDataProviderWrapper.tsx [hunk 5]
```diff
@@ -116,8 +122,8 @@ export const CloudDataProviderWrapper: React.FC<CloudDataProviderWrapperProps> =
   }, [roomsWithFullData, firebarrelsWithUsers, dataProvider]);
 
   useEffect(() => {
-    if (dataProvider) dataProvider.updateUsersAsync(worldUsers);
```
[the remark this answers is not in the document]
Comment (mike-lvov) [reply src/components/templates/AnimateMap/bridges/CloudDataProviderWrapper.tsx:119]: Yes, but by the beavers team

File: src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx [hunk 6]
```diff
@@ -1,31 +1,23 @@
...
-  const userLastSeenIn = userLocation?.lastVenueIdSeenIn;
-
-  const location = userLastSeenIn ? Object.keys(userLastSeenIn)?.[0] : "";
-
   return (
     <div key={user.id} className="RunTabUserInfo">
       <UserAvatar user={user} showStatus size="small" />
       <div className="RunTabUserInfo__user">
         <div>{user.partyName}</div>
-        {location && <div className="RunTabUserInfo__place">in {location}</div>}
```
[the remark this answers is not in the document]
Comment (mike-lvov) [reply src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx:28]: Planned to be restored, but I think what we should do here is just to reuse our profile modal

File: src/components/organisms/AdminVenueView/components/RunTabUsers/RunTabUsers.tsx [hunk 7]
```diff
@@ -19,25 +29,39 @@ interface RunTabSidebarProps {
...
+  const owners = venue?.owners ?? ALWAYS_EMPTY_ARRAY;
+
+  const [admins, setAdmins] = useState<WithId<User>[]>(ALWAYS_EMPTY_ARRAY);
+
+  useEffect(() => {
+    Promise.all(owners.map((owner) => getUserRef(owner).get())).then((docs) => {
+      const admins = docs.map((doc) => withId(doc.data() as User, doc.id));
+      setAdmins(admins);
+    });
+  }, [owners]);
```
[the remark this answers is not in the document]
Comment (mike-lvov) [reply src/components/organisms/AdminVenueView/components/RunTabUsers/RunTabUsers.tsx]: Good catch!. Fixed in https://github.com/sparkletown/sparkle/pull/2333/commits/3cd2fe9d1ff19d58a259d4749cc4ad93ac106d4d

**Pull request merged**